docker-compose.override.yml

```diff
@@ -75,6 +75,18 @@ services:
 ##      - emberosf
 ##    command: /bin/bash -c "cd /ember-osf && yarn link && cd /code && yarn link @centerforopenscience/ember-osf && yarn --pure-lockfile --ignore-engines && ./node_modules/bower/bin/bower install --allow-root --config.interactive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4300"
 
+  reviews:
+    #build: ../ember-osf-reviews # remove me before committing
```

Comment:
I believe this block needs to be commented out. Also, assuming this comment is correct, this line should be removed.